pkgs/build-support/docker/default.nix

```diff
@@ -864,13 +864,13 @@ rec {
         };
 
         closureRoots = lib.optionals includeStorePaths /* normally true */ (
-          [ baseJson ] ++ contentsList
+          [ baseJson customisationLayer ]
```

Comment:
`customizationLayer` includes the `contentsList`, so the closure of `closureRoots` is only extended with this line change.